docs/concepts.md

```diff
@@ -35,91 +46,102 @@ When there is late arriving data (data intended for 9:00 arriving >1 hr late at
+| Merge On Read | Read Optimized + Incremental + Near Real-time |
 
-Hudi storage types capture how data is indexed & laid out on the filesystem, and how the above primitives and timeline activities are implemented on top of
-such organization (i.e how data is written). This is not to be confused with the notion of Read Optimized & Near-Real time tables, which are merely how the underlying data is exposed
-to the queries (i.e how data is read).
+### Storage Types
+Hudi supports the following storage types.
 
-Hudi (will) supports the following storage types.
+  - [Copy On Write](#copy-on-write-storage) : Stores data using solely columnar file formats (e.g parquet). Updates simply version & rewrite the files by performing an synchronous merge during ingestion.
+  - [Merge On Read](#merge-on-read-storage) : Stores data using a combination of columnar (e.g parquet) + row based (e.g avro) file formats. Updates are logged to delta files & later compacted to produce new versions of columnar files synchronously or asynchronously.
+    
+Following table summarizes the trade-offs between these two storage types
 
-| Storage Type  | Supported Tables |
-|-------------- |------------------|
-| Copy On Write | Read Optimized   |
-| Merge On Read | Read Optimized + Near Real-time |
+| Trade-off | CopyOnWrite | MergeOnRead |
+|-------------- |------------------| ------------------|
+| Data Latency | Higher   | Lower |
+| Update cost (I/O) | Higher (rewrite entire parquet) | Lower (append to delta file) |
+| Parquet File Size | Smaller (high update(I/0) cost) | Larger (low update cost) |
+| Write Amplification | Higher | Lower (depending on compaction strategy) |
 
-  - Copy On Write : A heavily read optimized storage type, that simply creates new versions of files corresponding to the records that changed.
-  - Merge On Read : Also provides a near-real time datasets in the order of 5 mins, by shifting some of the write cost, to the reads and merging incoming and on-disk data on-the-fly
 
-Regardless of the storage type, Hudi organizes a datasets into a directory structure under a `basepath`,
-very similar to Hive tables. Dataset is broken up into partitions, which are folders containing files for that partition.
-Each partition uniquely identified by its `partitionpath`, which is relative to the basepath.
+### Views
+Hudi supports the following views of stored data
 
-Within each partition, records are distributed into multiple files. Each file is identified by an unique `file id` and the `commit` that
-produced the file. Multiple files can share the same file id but written at different commits, in case of updates.
+ - **Read Optimized View** : Queries on this view see the latest snapshot of the dataset as of a given commit or compaction action. 
+    The view exposes only the base/columnar files in latest file slices to the queries and guarantees the same columnar query performance compared to a non-hudi dataset. 
```

Comment:
`..guarantees the same raw columnar query performance compared to a non-hudi columnar format based dataset`